base/grafana/deploy-grafana.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -exuo pipefail
+command -v envsubst
+
+FILE_ROOT=${BASH_SOURCE%/*}
+NAMESPACE=${NAMESPACE:=tools}
+
+export FILE_ROOT NAMESPACE
+
+# make sure user-workloads are enabled
+USER_WORKLOADS=$(oc get cm/cluster-monitoring-config -n openshift-monitoring -ojsonpath='{.data.config\.yaml}')
+
+if ! [ "$USER_WORKLOADS" = "enableUserWorkload: true" ]; then
+    # maybe enable it here?
+    exit 0;
+fi
+
+envsubst < "${FILE_ROOT}"/operator-group.yaml.tpl | oc apply -n "${NAMESPACE}" -f -
+oc apply -f "${FILE_ROOT}"/subscription.yaml -n ${NAMESPACE}
+oc wait -n "${NAMESPACE}" --for=jsonpath=status.installPlanRef.name subscription grafana-operator --timeout=120s
+oc wait -n "${NAMESPACE}" --for=condition=Installed installplan --all --timeout=120s
```

Comment:
you can wait for only this installplan to be installed through the reference in the subscription
`kubectl wait -n "${NAMESPACE}" installplan "$($CTL get -n "${NAMESPACE}" subscription grafana-operator -o=jsonpath='{.status.installPlanRef.name}')" --for=condition=Installed --timeout="$TIMEOUT_TIME"s`